src/test/scala/mesosphere/marathon/core/externalvolume/impl/providers/DVDIProviderRootGroupValidationTest.scala

```diff
@@ -74,8 +74,33 @@ class DVDIProviderRootGroupValidationTest extends UnitTest with GroupCreation {
       )
     }
 
+    "two volumes with same name and shared true result in no error" in {
+      val f = new Fixture
+      Given("a root group with two apps and conflicting volumes")
+      val app1 = f.appWithDVDIVolume(appId = PathId("/nested/app1"), volumeName = "vol", shared = true)
+      val app2 = f.appWithDVDIVolume(appId = PathId("/nested/app2"), volumeName = "vol", shared = true)
+      val rootGroup = createRootGroup(
+        groups = Set(
+          createGroup(
+            id = AbsolutePathId("/nested"),
+            apps = Map(
+              app1.id -> app1,
+              app2.id -> app2
+            ),
+            validate = false
+          )
+        ),
+        validate = false
+      )
+
+      f.checkResult(
+        rootGroup,
+        expectedViolations = Set.empty
```

Comment:
`expectedViolations` is empty? Shouldn't this have a violation error?

Reply:
No, this test verifies that `shared = true` actually works. It creates two apps with the same volume name, which should be accepted if they have the `shared` flag set to true